catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswUnmarshallHelper.java

```diff
@@ -76,7 +77,7 @@ public static Date convertToDate(String value) {
         // the time zone and time are optional.
```

Comment:
Sounds like this comment is no longer correct. Is UTC always the correct time zone? Do you need to parse first to determine if it is local and then fall back on UTC otherwise?